index.html

```diff
@@ -97,7 +97,122 @@ <h2>Software Developer</h2>
+                        since the 1500s, when an unknown printer took a standard dummy text.
+                    </p>
+                    <ul class="workTechs">
+                        <li>css</li> 
+                        <li>html</li>
+                        <li>Bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                    <button class="seebutton">See project</button>
+
+                </li>
+                <li class="workCard">
+                    <img src="Images/projectOne.png" alt="projectone">
+                    <h3>Multi-post Stories</h3>
+                    <p>A daily selection of privately personalized reads; no accounts or
+                        sign-ups required. has been the industry's standard dummy text ever
+                        since the 1500s, when an unknown printer took a standard dummy text.
+                    </p>
+                    <ul class="workTechs">
+                        <li>css</li> 
+                        <li>html</li>
+                        <li>Bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                    <button class="seebutton">See project</button>
+
+                </li>
+                
+               
+            </ul>
+            <section class="contactus">
+                <h2 class="contactustitle">CONTACT US</h2>
+                <P>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </P>
+              
+                <form action="/Action" method="post" name="contactus" class="contactusform">
+                    <input type="text" name="Fullname" placeholder="Full name">
+                    <input type="email" name="email" placeholder="E-mail">
+                    <textarea placeholder="Enter text here"> 
+                    </textarea>
+                    <button type="submit" class="seebutton"> Send</button>
+                </form> 
+
+            </section>
+
+        </section>
     </main>
+    <footer class="footer">
+        <ul class="socialmedia">
+            <li>
+                <a href="">
+                    <img src="images/icons/github.png" alt="github">
+                </a>
+            </li>
+            <li>
+                <a href="">
+                    <img src="images/icons/linkedin.png" alt="peace">
+                </a>
+
+            </li>
+            <li>
+                <a href="">
+                    <img src="images/icons/peace.png" alt="peace">
+                </a>
+            </li>
+            <li>
+                <a href="">
+                    <img src="images/icons/twitter.png" alt="peace">
+                </a>
+            </li>
+            <li>
+                <a href="">
+                    <img src="images/icons/m.png" alt="M">
+                </a>
+            </li>
+        </ul> 
+    </footer> 
```

Comment:
### Your Project :bust_in_silhouette:

![about section icons](https://user-images.githubusercontent.com/93039359/176910280-d547ecc5-4060-450d-a4b6-9332395d0195.png)

## 1.
- [ ] Please fix your styling because you have horizontal scrollbar. Follow the steps to fix the width :+1:
### Step 1
![styling changes](https://user-images.githubusercontent.com/93039359/176910597-4558794d-d9bf-4461-97d4-13d8d2677f67.png)

### Step 2
![styling part 2](https://user-images.githubusercontent.com/93039359/176910619-8a1ec82f-ba2e-4d1a-8fe0-324b2cbd2916.png)

## 2.
- [ ] Your icons are not visible please fix the URLs. :+1: